tests/helpers/patch_disco_aws.py

```diff
@@ -57,7 +57,8 @@ def get_default_config_dict():
                           "logforwarder_hostclass": "mhclogforwarder",
                           "default_smoketest_termination": "True",
                           "default_environment": "auto-vpc-type",
-                          "default_domain_name": "example.com"},
+                          "default_domain_name": "example.com",
+                          "default_spotinst_account_id": "foo"},
```

Comment:
I prefer something that looks more obvious when we get an error. Like
```suggestion
                          "default_spotinst_account_id": "fake_spotinst_account_id"},
```